nemo/backends/pytorch/actions.py

```diff
@@ -1252,67 +1340,70 @@ def train(
                 else:
                     train_sampler = None
 
-            for train_iter in training_loop:
-                call_chain = train_iter[2]
-                for i in range(1, len(call_chain) - 1):
-                    key = call_chain[i][0].unique_instance_id
-                    pmodule = self.module_reference_table[key][1]
-                    num_trainable_weights = self.module_reference_table[key][1].num_weights
-                    if (
-                        not isinstance(pmodule, DDP)
-                        and isinstance(pmodule, torch.nn.Module)
-                        and num_trainable_weights > 0
-                    ):
-                        # gpf = 1
-                        # if gradient_predivide:
-                        #     gpf = dist.get_world_size()
-                        # pmodule = DDP(pmodule, gradient_predivide_factor=gpf)  # Old Apex Method
-
-                        # Per pytorch docs, convert sync bn prior to DDP
-                        if synced_batchnorm:
-                            world_size = dist.get_world_size()
-                            sync_batchnorm_group = None
-                            if synced_batchnorm_groupsize > 0:
-                                if world_size % synced_batchnorm_groupsize != 0:
-                                    raise ValueError(
-                                        f"Synchronized batch norm group size ({synced_batchnorm_groupsize}) must be 0"
-                                        f" or divide total number of GPUs ({world_size})."
-                                    )
-                                # Find ranks of other nodes in the same batchnorm group
-                                rank = torch.distributed.get_rank()
-                                group = rank // synced_batchnorm_groupsize
-                                group_rank_ids = range(
-                                    group * synced_batchnorm_groupsize, (group + 1) * synced_batchnorm_groupsize
+            # for train_iter in training_loop:
+            #     call_chain = train_iter[2]
+            #     for i in range(1, len(call_chain) - 1):
+            #         key = call_chain[i][0].unique_instance_id
+            #         pmodule = self.module_reference_table[key][1]
+            #         num_trainable_weights = self.module_reference_table[key][1].num_weights
```

Comment:
There is plenty of those commented blocks... Why are we leaving them? What is our intention here?

Reply:
Did a scrub over the files in this PR and removed commented code.